src/endpoints/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+option(IRODS_ENABLE_GENQUERY2 "Enables GenQuery2 support. GenQuery2 package must be installed." NO)
```

Comment:
where?   here?   in the iRODS server?

Reply:
GenQuery2 must be installed on the computer used to compile the HTTP API (if they want it).

GenQuery2 must be installed on the iRODS server which the HTTP API is connected to (if the GenQuery2 API plugin is going to be invoked).

Reply:
so, both.   okay.    that wasn't clear in my mind - so we need some more notes here or in build section of readme too perhaps.

Reply:
Added words to README.